src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle() {
+    }
+
+    public RightTriangle(double firstLeg, double secondLeg) {
+        this.typeOfFigure = TypeOfFigure.RIGHT_TRIANGLE;
+        this.area = getTriangleArea(firstLeg, secondLeg);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: " + typeOfFigure
+                + ", area: " + decimalFormat.format(area) + " sq. units, first leg: "
+                + decimalFormat.format(firstLeg) + " units, second leg: "
+                + decimalFormat.format(secondLeg) + " units, color: " + color);
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition (line 2) and the `drawFigure` method (line 17).